core/src/test/java/org/bitcoinj/evolution/QuorumStateValidateQuorumsTest.java

```diff
@@ -81,14 +83,16 @@ public static Collection<Object[]> data() {
                         "mnlistdiff-testnet-0-850798-70228-after19.2HF.dat",
                         "testnet-after19.2HF.spvchain",
                         70228,
-                        850798
+                        850798,
+                        SimplifiedMasternodeListManager.SMLE_VERSION_FORMAT_VERSION
```

Comment:
These tests check that we would be saving in the lastest format version.

In the future, I was wondering if the save format should be in protobuf instead of the bitcoin wire protocol format which is easy to mess up with changes from one version to the next.